### DISCUSSION TITLE
我上架了Firefox，暂时没被下架……

### DISCUSSION THREAD

**Discussion by Leojc** [ronggang/PT-Plugin-Plus#1302] [Show and tell]
需要可以使用，绝无自己修改（我也不会……），就是原版加上增强补丁。
https://addons.mozilla.org/zh-CN/firefox/addon/pt-plugin-plus-pt%E5%8A%A9%E6%89%8B%E7%99%BD%E5%A4%A7%E5%A2%9E%E5%BC%BA%E7%89%88/

**Comment by Leojc** [comment]
更新1.6.1.1994

**Reply by ghost** [reply]
请问有时间可以更新一下吗，似乎有点功能用不了

**Reply by Leojc** [reply]
现在好像可以直接装xpi文件的
